lmd/objects.go

```diff
@@ -616,11 +616,13 @@ func NewHostsTable() (t *Table) {
 	t.AddColumn("notification_interval", StaticUpdate, IntCol, "Interval of periodic notification or 0 if its off")
 	t.AddColumn("notification_period", StaticUpdate, StringCol, "Time period in which problems of this host will be notified. If empty then notification will be always")
 	t.AddColumn("notifications_enabled", DynamicUpdate, IntCol, "Whether notifications of the host are enabled (0/1)")
+	t.AddColumn("obsess", DynamicUpdate, IntCol, "The obsessing over host")
```

Comment:
better add new Columns at the end (right before the AddOptColumn...). Otherwise we have to update the test data as well.

Reply:
Updated. Thanks. But still benchmark_test.go fails.
--- FAIL: BenchmarkParseResultJSON
	benchmark_test.go:37: wrong result size, expected 181, got 185
--- FAIL: BenchmarkParseResultWrappedJSON
	benchmark_test.go:75: wrong result size, expected 181, got 185
Do I have to update this test case to 185?

Reply:
exactly. btw, you can run the travis locally by `make citest`

Reply:
Yah sure. Thanks.